tests/unit_tests/test_wave_06.py

```diff
@@ -75,17 +75,20 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    # Assert
+    assert result
+    assert len(tai.inventory) == len(jesse.inventory) == 3
+    assert item_f in tai.inventory 
+    assert item_c in jesse.inventory
+    assert item_c not in tai.inventory
+    assert item_f not in jesse.inventory
```

Comment:
It's great that the asserts check on the items that were swapped, but we should add asserts about the other items in the lists too, to make sure there were no unexpected side affects.